src/dotnet-install.ps1

```diff
@@ -340,9 +340,8 @@ function Get-Latest-Version-Info([string]$AzureFeed, [string]$Channel) {
     elseif ($Runtime -eq "aspnetcore") {
         $VersionFileUrl = "$UncachedFeed/aspnetcore/Runtime/$Channel/latest.version"
     }
-    # Currently, the WindowsDesktop runtime is manufactured with the .Net core runtime
     elseif ($Runtime -eq "windowsdesktop") {
-        $VersionFileUrl = "$UncachedFeed/Runtime/$Channel/latest.version"
+        $VersionFileUrl = "$UncachedFeed/WindowsDesktop/$Channel/latest.version"
```

Comment:
Are you sure that for all channels "Runtime" should be changed to "WindowsDesktop" in this path? I thought it was only for 5.0.

Reply:
Ah, I see that you'd like to move the files. That is strange for me that for channels below 5 the information will be placed in two different places: some path contains "/Runtime/" and some "/WindowsDesktop/". I'd vote for adding a check here rather than moving files.

Reply:
https://github.com/dotnet/install-scripts/pull/120#issuecomment-740042805 
Matt mentioned that they will moved the files. Also there is a plan to move all the packages to /WindowsDesktop/.

Reply:
Yeah, I saw this comment too late. Well, if there are plans to move all the packages to /WindowsDesktop/, then I suppose it makes sense to move some things right now.